crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -328,12 +328,16 @@ spec:
                       required: ["name", "mount"]
                       type: object
                       properties:
-                        name:
-                          description: "The name of the secret that is to be mounted to the Kiali pod's file system. The name of the custom secret must not be the same name as one created by the operator. Names such as `kiali`, `kiali-cert-secret`, and `kiali-cabundle` should not be used as a custom secret name because the operator may want to create one with one of those names."
-                          type: string
+                        csi:
+                          description: "Defines CSI-specific settings that allows a secret from an external CSI secret store to be injected in the pod via a volume mount. For details, see https://secrets-store-csi-driver.sigs.k8s.io/"
```

Comment:
@rumstead can you review this little documentation blurb? This will show up in the kiali.io documentation website in the Kiali CR reference page - it will end up somewhere around [here](https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.deployment.custom_secrets[*]). I just want to make sure I'm not blowing smoke :) If you have some better documentation blurb to put here, please suggest it.

Reply:
LGTM. Should we add an example to the already [existing examples](https://github.com/kiali/kiali-operator/blob/87e66d115e854166869f2cb8c272a2207ec325a3/crd-docs/crd/kiali.io_kialis.yaml#L319)?

Maybe:
```yaml
    - name: "kiali-secret-csi"
      mount: "/mnt/secrets-store"
      csi:
        driver: secrets-store.csi.k8s.io
        readOnly: true
        volumeAttributes:
          secretProviderClass: kiali-secretprovider
```

 If you think we need more docs in the future like there are for custom dashboards I can [contribute something](https://kiali.io/docs/configuration/custom-dashboard/).

Reply:
> LGTM. Should we add an example to the already [existing examples](https://github.com/kiali/kiali-operator/blob/87e66d115e854166869f2cb8c272a2207ec325a3/crd-docs/crd/kiali.io_kialis.yaml#L319)?

We could. I can add it to this PR. I don't know what the true values would/should look like, so please suggest something that would be good for that example CR.

> If you think we need more docs in the future like there are for custom dashboards I can [contribute something](https://kiali.io/docs/configuration/custom-dashboard/).

Yes. I was thinking we should have a dedicated "something" in the docs for this. I was thinking it is a simple FAQ that we would add [here](https://github.com/kiali/kiali.io/blob/staging/content/en/docs/FAQ/installation.md?plain=1) which shows up [here](https://kiali.io/docs/faq/installation/). Feel free to submit a PR over in the [kiali.io repo](https://github.com/kiali/kiali.io).